.circleci/config.yml

```diff
@@ -51,18 +47,22 @@ jobs:
       - disable_docker_hub
       - attach_workspace:
           at: /tmp/workspace
-
-      # - configure_environment
       - checkout
       - python/install-packages:
           pkg-manager: pip
       - run:
           name: Run Unit Tests
           command: |
             set -x
+            export ENV_VAR_TENANT=${ENV_VAR_TENANT}
+            export ENV_VAR_USERNAME=${ENV_VAR_USERNAME}
+            export ENV_VAR_PASSWORD=${ENV_VAR_PASSWORD}
+            export ENV_VAR_API_KEY=${ENV_VAR_API_KEY}
+            export ENV_VAR_API_SECRET=${ENV_VAR_API_SECRET}
```

Comment:
Are these not already exported environment variables?  Echoing sensitive data such as password/secrets is bad.  If needed, better to move the `set -x` later.

Reply:
"You are correct. As I've already added these new environment variables to the project's context, there is no need to export them here. However, CircleCI is smart enough not to print this sensitive information, even if we try to display it.

I have removed these export statements and updated the 'config.yml'."